deployment/v3/utils/prop_migrator/knowledge/new-property-with-decent-default-value.csv

```diff
@@ -1623,3 +1623,27 @@ resident-default.properties,mosip.scope.resident.postRequestShareCredWithPartner
 resident-default.properties,resident.template.support-docs-list
 resident-default.properties,resident.identity.info.version
 resident-default.properties,resident.template.email.content.failure.cust-and-down-my-card
+packet-manager-default.properties,auth.server.admin.allowed.audience
+packet-manager-default.properties,mosip.kernel.idobjectvalidator.referenceValidator
+packet-manager-default.properties,packetmanager.additional.fields.search.from.metainfo
+packet-manager-default.properties,mosip.role.commons-packet.putcreatepacket
+packet-manager-default.properties,mosip.role.commons-packet.postaddtag
+packet-manager-default.properties,mosip.role.commons-packet.postaddorupdatetag
+packet-manager-default.properties,mosip.role.commons-packet.postdeletetag
+packet-manager-default.properties,mosip.role.commons-packet.postinfo
+packet-manager-default.properties,mosip.role.commons-packet.postgettags
+packet-manager-default.properties,mosip.role.commons-packet.postvalidatepacket
+packet-manager-default.properties,mosip.role.commons-packet.postaudits
+packet-manager-default.properties,mosip.role.commons-packet.postmetainfo
+packet-manager-default.properties,mosip.role.commons-packet.postbiometrics
+packet-manager-default.properties,mosip.role.commons-packet.postdocument
+packet-manager-default.properties,mosip.role.commons-packet.postsearchfields
+packet-manager-default.properties,object.store.s3.accesskey
+packet-manager-default.properties,object.store.s3.secretkey
```

Comment:
looks like a duplicate, can we remove one ?

Reply:
no duplicate present,I verified